bindings/go/src/fdb/fdb_test.go

```diff
@@ -193,25 +210,33 @@ func ExampleReadTransactor() {
 
 	getOne := func(rt fdb.ReadTransactor, key fdb.Key) ([]byte, error) {
 		fmt.Printf("getOne called with: %T\n", rt)
-		ret, err := rt.ReadTransact(func(rtr fdb.ReadTransaction) (interface{}, error) {
+		ret, txClose, err := rt.ReadTransact(context.Background(), func(rtr fdb.ReadTransaction) (interface{}, error) {
```

Comment:
Shouldn't we call in the method body `defer .Close()` on the returned future from the `rtr.Get` method? Otherwise we have a text example that introduces a memory leak?